about.html

```diff
@@ -13,51 +13,80 @@
-
-
+      <!--header with all Lambda logo and Navbar included-->
+      <article class="header">
+        <img src="img/lambda-black.png" alt="lambda-logo">
+        <nav class="navbar">
+           <a href="index.html">Home</a>
+           <a href="about.html"> About</a>
+           <a href="#">Products</a>
+           <a href="#">Blog</a>
+           <a href="#">Contact</a>
+        </nav>
+       </article>
+       <!--About section since it was different then the other sections-->
+       <img src="img/jumbo-about.png" alt="city-view-skyline">
+     <article class="aboutSection">
+    <h4>About</h4>
+
+    <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. 
+      Iterative approaches to corporate strategy foster collaborative thinking to further the 
+      overall value proposition. Organically grow the holistic world view of disruptive innovation 
+      via workplace diversity and empowerment.</p>
+
+    </article>
+    <!--First section, I wanted to divede the photos in two sections to make it easier to move-->
+      <section class="firstSection">
+        <div class="top1">
     <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
+    <h3>Strategy</h3>
 
+    <p>Iterative approaches to corporate strategy foster collaborative thinking to further 
+    the overall value proposition.</p>
 
+    <button onclick="window.location.href='#';"> Learn More</button>
+        </div>
+        <div class="top2">
     <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
+    <h3>How We Work</h3>
 
+  <p>Iterative approaches to corporate strategy foster collaborative thinking to further the 
+    overall value proposition.</p>
 
+    <button onclick="window.location.href='#';"> Learn More</button>
+  </div>
+     </section>
+<!--Second section, I wanted to divede the photos in two sections to make it easier to move-->
+     <section class="secondSection">
+     <div class="bottom1"> 
     <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
 
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+    <h3>Places We Work</h3>
 
-    Learn More
+    <p>Iterative approaches to corporate strategy foster collaborative thinking to further 
+    the overall value proposition.</p>
 
+    <button onclick="window.location.href='#';"> Learn More </button>
+  </div>
+    <div class="bottom1">
```

Comment:
Don't forget to change the alt="" to match the section you are in.  All of them for these for sections say "strategy"